src/test/java/org/fundacionjala/coding/escarleth/Kata8Test.java

```diff
@@ -0,0 +1,79 @@
+package org.fundacionjala.coding.escarleth;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * This class contains UnitTests for Kata8.
+ */
+
+public class Kata8Test {
+    private Kata8 kata8;
+    private static final String COMMON_LINE1 = " _  _     _  _        _  _ ";
+    private static final String COMMON_LINE2 = "    _  _     _  _  _  _  _ ";
+    private static final String COMMON_LINE3 = "  |  |  |  |  |  |  |  |  |";
+
+    /**
+     * initialize kata8.
+     */
+    @Before
+    public void initialize() {
+        kata8 = new Kata8();
+    }
+
+    /**
+     * BankKata test1 expected 123456789.
+     */
+    @Test
+    public void test11BankKata8() {
+        String firstRow = "    _  _     _  _  _  _  _ ";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'COMMON_LINE2' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)